flixel/effects/postprocess/PostProcess.hx

```diff
@@ -61,21 +62,21 @@ class PostProcess extends OpenGLView
 
 		buffer = GL.createBuffer();
 		GL.bindBuffer(GL.ARRAY_BUFFER, buffer);
-		GL.bufferData(GL.ARRAY_BUFFER, new Float32Array(cast vertices), GL.STATIC_DRAW);
+		GL.bufferData(GL.ARRAY_BUFFER, new Float32Array(#if !openfl_next cast #end vertices), GL.STATIC_DRAW);
```

Comment:
>OpenFL next doesn't need to cast.

Does it really matter to have the `cast` there in next anyway though? Unsafe casts shouldn't have much of a runtime overhead.